transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/TransformationPipelineBuilder.scala

```diff
@@ -36,11 +36,47 @@ object TransformationPipelineBuilder extends PipelineBuilder[Args] {
       .map(DonorTransformations.transformDonor)
     StorageIO.writeJsonLists(donorOutput, "Donors", s"${args.outputPrefix}/donor")
 
-    // So can antibodies.
+    // The Antibody transformation needs information from the Target objects
     val antibodyInputs = readRawEntities(EncodeEntity.AntibodyLot)
+    val targetInputs = readRawEntities(EncodeEntity.Target)
+    val keyedTargets = targetInputs
+      .withName("Key targets by id")
+      .keyBy(_.read[String]("@id"))
+
+    // Get all of the (Antibody-id, Target-id) pairs
+    val antibodyTargetPairs = antibodyInputs.map { rawAntibody =>
```

Comment:
Using a `flatMap` here would be more idiomatic than a `map` followed by `flatten`. It might be more efficient at runtime, too.

Reply:
Can we also add a `withName` after `antibodyInputs` to describe the operation?

Reply:
+1 to using `flatMap`

Reply:
Just updated 👍